cmlreaders/__init__.py

```diff
@@ -1,7 +1,7 @@
 from collections import namedtuple
 from .path_finder import *  # noqa
 from .readers import * # noqa
-from .cmlreader import CMLReader
+from .cmlreader import * # noqa
```

Comment:
I'd rather be explicit here even if `__all__` is defined. Using `*` imports should be restricted to cases where there are lots of things to import.